docs/developers/contributing/documentation/index.md

```diff
@@ -115,6 +115,16 @@ want to contribute. The paths are listed in parentheses below.
 - [**Getting started**](getting-started) (in [`napari/docs/tutorials/fundamentals`](https://github.com/napari/docs/tree/main/docs/tutorials/fundamentals)):
   these documents are a mix of tutorials and how-tos covering the fundamentals of installing and working with napari for beginners
 
+```{admonition} Plugin documentation
+:class: tip
+
+Some of the source files for the [Plugin documentation](plugins-index) are
+autogenerated from sources in the [napari/npe2](https://github.com/napari/npe2)
+repository. These files' names start with an underscore `_` and will be
```

Comment:
Could we add an example, "e.g., `_npe2_contributions.md`"

Reply:
Or even add that these added files all live in `docs/plugins/` (I think this is right?)

Reply:
I was hesitant to mention filenames and location because these may change and keeping this document up-to-date may be difficult (for example we recently moved a bunch of files under the plugins folder). But I think mentioning they _are_ under "plugins" is reasonable indeed.